sway/security.c

```diff
@@ -94,7 +141,32 @@ static const char *get_pid_exe(pid_t pid) {
 	return link;
 }
 
-uint32_t get_feature_policy(pid_t pid) {
+struct feature_policy *get_feature_policy(const char *name) {
+	struct feature_policy *policy = NULL;
+	char *program = NULL;
+
+	if (!(program = resolve_path (name))) {
+		sway_abort ("Unable to allocate security policy");
+	}
+	for (int i = 0; i < config->feature_policies->length; ++i) {
+		struct feature_policy *p = config->feature_policies->items[i];
+		if (strcmp (p->program, program) == 0) {
+			policy = p;
+			break;
+		}
+	}
+	if (!policy) {
+		policy = alloc_feature_policy (program);
+		if (!policy) {
+			sway_abort ("Unable to allocate security policy");
+		}
+		list_add (config->feature_policies, policy);
```

Comment:
Muscle memory. Until Sway becomes my focus project, my default clang-format belongs to someone else. 

Fixed.